tools/cockpit.spec.in

```diff
@@ -155,6 +152,7 @@ Recommends: subscription-manager-cockpit
 
 %build
 %configure \
+    %{?selinux_configure_arg} \
```

Comment:
This could even move here statically, as it would be a no-op for c-appstream.

Reply:
In the sense that we could build and install the file and then just ignore it when building the package lists?

Reply:
As per IRC: This is to prevent having to remove the uninstalled files, otherwise rpmbuild complains. So this is easier after all.